editor/editor_node.cpp

```diff
@@ -1444,6 +1437,19 @@ void EditorNode::_sources_changed(bool p_exist) {
 		if (!singleton->cmdline_mode) {
 			EditorResourcePreview::get_singleton()->start();
 		}
+
+		// Set initial focus for screen reader users.
+		if (get_tree()->is_accessibility_enabled()) {
+			TabContainer *tab_container = EditorDockManager::get_singleton()->get_dock_tab_container(SceneTreeDock::get_singleton());
+			if (tab_container && tab_container->get_current_tab_control() == SceneTreeDock::get_singleton()) {
+				// Scene dock is active - focus the scene tree.
+				SceneTreeDock::get_singleton()->get_tree_editor()->get_scene_tree()->grab_focus();
+			} else if (tab_container) {
+				// Another tab is active (e.g., Import) - focus the tab bar so user can switch.
+				tab_container->get_tab_bar()->grab_focus();
+			}
```

Comment:
This can be simplified to (not tested)

```suggestion
			if (SceneTreeDock::get_singleton()->is_visible_in_tree()) {
				SceneTreeDock::get_singleton()->get_tree_editor()->get_scene_tree()->grab_focus();
			} else {
				TabContainer *tab_container = EditorDockManager::get_singleton()->get_dock_tab_container(SceneTreeDock::get_singleton());
				if (tab_container) {
					// Another tab is active (e.g., Import) - focus the tab bar so user can switch.
					tab_container->get_tab_bar()->grab_focus();
				}
			}
```
Note that if the Scene dock is missing for whatever reason (like closed or something), the editor will still have no focus. It's unlikely to happen though.